packages/component-library/stories/Button.story.js

```diff
@@ -3,34 +3,103 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 import { checkA11y } from "@storybook/addon-a11y";
-import { withKnobs, text } from "@storybook/addon-knobs";
+import {
+  withKnobs,
+  text,
+  color,
+  optionsKnob as options,
+  select
+} from "@storybook/addon-knobs";
+import notes from "./button.markdown.md";
 import { Button } from "../src";
 import { storybookStyles } from "./storyStyles.js";
 
-const displayName = Button.displayName || "Button";
-
-const description = `
-  This is some basic usage with the button with providing a label to show the text.
-  Clicking should trigger an action.`;
-
-const simpleButtonLabel = "Hello button";
-
 export default () =>
-  storiesOf("Component Lib|Basic UI Components/Button", module)
+  storiesOf("Component Lib|Basic Inputs/Button", module)
     .addDecorator(withKnobs)
     .addDecorator(checkA11y)
     .addDecorator(story => (
       <div style={storybookStyles.storyGrid}>
         <div style={storybookStyles.storyGridItem}>{story()}</div>
       </div>
     ))
-    .add("Simple usage", () => (
-      <Button onClick={action("clicked")}>{simpleButtonLabel}</Button>
-    ))
-    .add("With some emoji", () => (
-      <Button onClick={action("clicked")}>
-        <span role="img" aria-label="So cool">
-          😀 😎 👍 💯
-        </span>
-      </Button>
-    ));
+    .add(
+      "Standard",
+      () => {
+        const label = text("Label", "Hello CIVIC");
+        return <Button onClick={action("clicked")}>{label}</Button>;
+      },
+      { notes }
+    )
+    .add(
+      "Custom",
+      () => {
+        const accentColor = color("Text & border color", "#DC4556");
+        const bkgndColor = color("Background color", "#FFFFFF");
+        const label = text("Label", "Hello CIVIC");
+        const transitionOptions = {
+          Standard: "all .2s ease-in-out",
+          Slow: "all .6s ease-in-out",
+          Fast: "all .1s ease-in-out"
+        };
+        const transition = select(
+          "Transition",
+          transitionOptions,
+          "all .2s ease-in-out"
+        );
+        //const disabled = boolean("Disabled", false);
+        const valuesObj = {
+          Enabled: "",
+          Disabled: "disabled"
+        };
+        const optionsObj = {
+          display: "inline-radio"
+        };
+        const disabled = options("State", valuesObj, "", optionsObj);
+        //console.log(disabled);
```

Comment:
Per above, recommend pulling the knobs outside of the function block body so they can be reused across stories.

Reply:
As notes above, doesn't work